index.html

```diff
@@ -212,72 +224,76 @@
+              {
+                exportDivId : "#export-div"
+              }), 
+            enabled: true}
+        ])
         .collapseAll();
       timeline();
-
-      $("#trim").prop("checked", true)
-
-      //initiliase simple-advanced buttons
-      if (enableAdvanced) {
-        $("#advBtn").prop("disabled", true); //disable repeated clicking of advButton
-        $("#advBtn").addClass("active");
-        $("#simpleBtn").prop("disabled", false); //enable simpleBtn
-        $("#simpleBtn").removeClass("active");
-        $(".dropdown-toggle").prop("disabled", true);
-        $("#jsonInput").css("display", "block");
-        $("#tooltip-controller").css("visibility", "hidden");
-        $(".radio").css("visibility", "hidden");
-        if (location.href.indexOf("?view=simple") > -1) {  
-          window.history.replaceState('advancedView', '', location.href.replace("?view=simple", "?view=advanced"));
-        }
-      } else {
-        $("#advBtn").prop("disabled", false); //enable advBtn
-        $("#advBtn").removeClass("active");
-        $("#simpleBtn").prop("disabled", true); //disable repeated clicking of simpleBtn
-        $("#simpleBtn").addClass("active");
-        $(".dropdown-toggle").prop("disabled", false);
-        $("#jsonInput").css("display", "none");
-        $("#tooltip-controller").css("visibility", "visible");
-        $(".radio").css("visibility", "visible");
-        if (location.href.indexOf("?view=advanced") > -1) {  
-          window.history.replaceState('simpleView', '', location.href.replace("?view=advanced", "?view=simple"));
-        }
-      }
     }
   </script>
 </head>
 <body>
-  <div>
+  <div id="timeline-wrapper">
     <h3>clinical timeline</h3>
-    <div class="dropdown">
-      <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown">Export As
+    <div class="dropdown" id="config-div">
+        <button class="btn btn-default dropdown-toggle" type="button" id="configDropdownBtn" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+          Config
+          <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu" id="config-dropdown">
+          <!-- dynamically populated via js/plugins/configButton.plugins -->
+        </ul>
+    </div>
+    <div class="dropdown" id="export-div">
+      <button class="btn btn-default dropdown-toggle" id="export-button" type="button" data-toggle="dropdown">Export As
       <span class="caret"></span></button>
       <ul class="dropdown-menu">
-        <li><a href="javascript:clinicalTimelineExporter.generateSVG()">SVG Image</a></li>
-        <li><a href="javascript:clinicalTimelineExporter.generatePNG(true)">PNG Image</a></li>
-        <li><a href="javascript:clinicalTimelineExporter.generatePDF()">PDF Document</a></li>
+        <li><a href='javascript:(new clinicalTimelineExporter()).run(null, {exportDivId : "#export-div"}).generateSVG()'>SVG Image</a></li>
```

Comment:
This is fine, but I think it would make more sense to only create the `clinicalTimelineExporter` once. You pass the constructor the `#export-div` config there. That can be saved on the object. You would then call generateSVG on that object. No need to pass `#export-div` twice and create two `clinicalTimelineExporter` instances.